### PR TITLE
#586/If a user is playing add a way to navigate to their active game from their user page

### LINKED ISSUE #586: If a user is playing, add a way to navigate to her/his active game from her/his user page
> podríamos cambiar el botón de retar al jugador por ir a live game cuando esto ocurra
> 
> _Originally posted by @vdedios [here](https://42born2code.slack.com/archives/C033YQ28QA0/p1678446800616739?thread_ts=1678445978.081839&cid=C033YQ28QA0)

### PR DESCRIPTION
**Pull request by gbudau**
close #586

### CODE REVIEW

File: webapp/src/pages/UserPage/UserPage.tsx [hunk 1]
```diff
@@ -152,6 +165,17 @@ export default function UserPage() {
...
+          (!overridePermissions &&
+            user &&
+            user.isFriend !== null &&
+            userStatus(user.id) === 'playing' && {
+              variant: ButtonVariant.ALTERNATIVE,
+              iconVariant: IconVariant.PLAY,
+              onClick: () => {
+                socket.emit('getPlayerGame', { playerId: user.id });
+              },
+              children: 'spectate player',
```
Comment (Damrod) [inline webapp/src/pages/UserPage/UserPage.tsx:177]: What do you think about 'watch live game' instead of 'spectate game'?

**Review by Damrod — APPROVED** [review]
Lgtm!

File: webapp/src/pages/UserPage/UserPage.tsx (hunk 1 above)
Comment (gbudau) [reply webapp/src/pages/UserPage/UserPage.tsx]: I've updated it. Thanks for the suggestion.

**Comment by gbudau** [comment]
Thanks!

**Pull request merged**